CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ For migration information, you can always have a look at https://liip-drifter.re
 
 ### Changed
 
+- Nginx role: added Drupal templates (D6, D7 and D8)
```

Comment:
Moved the changelog to the correct location, plus added description into `webservers.md`